tests/index.js

```diff
@@ -1,40 +1,49 @@
 const dotenv = require('dotenv')
 const sinon = require('sinon')
+const tap = require('tap')
 const loadConfiguration = require('../index')
 
-exports.loadConfiguration = {
-  setUp: function (done) {
+tap.test('loadConfiguration', (module) => {
+  let configStub
```

Comment:
Put config stub in the `beforeEach` as `configStub = sinon.stub(dotenv, 'config')`.  The `afterEach` will fail if it's not defined.